### ISSUE TITLE
2x 3080ti's going FHR when in good temperatures.

### ISSUE THREAD

**Issue by Faltskog** [trexminer/T-Rex#1167]
Don't know if this a common topic or something wrote about this, but i think LHR hits only when GPU's registers temperature, therefore tiny thermal throttles and i think that could be the solution.
 
Is there's a way to bypass temperature/thermal throttle? Override overheat? Ignore temp signals? 

It also seems that thermal throttling is escalonated and are multiple of them which they hit a registry within GPU and sends signal to downgrade it's own performance, thus happens THIS.

![WhatsApp Image 2022-03-16 at 05 55 00](https://user-images.githubusercontent.com/90661265/158603896-7d3cbd0a-f4e4-4f79-a943-aa060cb919cc.jpeg)

**Comment by Faltskog** [comment]
@trexminer

**Comment by CryptoDonkeyMiner** [comment]
Nah, looking at your uptime it is because LHR needs time to kick in.

**Comment by Faltskog** [comment]
@CryptoDonkeyMiner but im mining over a year and never register that FHR number, only when is on good temps.
